src/web/components/Nav/ExpandedMenu/ExpandedMenuToggle/NoJSButton.tsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import { css } from 'emotion';
+
+import { from } from '@guardian/src-foundations/mq';
+import { headline } from '@guardian/src-foundations/typography';
+import {
+    brandText,
+    brandAlt,
+    neutral,
+} from '@guardian/src-foundations/palette';
+
+type Props = {
+    menuCheckboxId: string;
+    display: Display;
+    isVeggieBurger: boolean;
+    children: JSX.Element | JSX.Element[];
+};
+
+const veggieBurgerStyles = (display: Display) => css`
+    background-color: ${brandAlt[400]};
+    color: ${neutral[7]};
+    cursor: pointer;
+    height: 42px;
+    min-width: 42px;
+    position: absolute;
+    border: 0;
+    border-radius: 50%;
+    outline: none;
+
+    /* TODO: we should not use such a hight z-index number  */
+    z-index: 1071;
+
+    right: 5px;
+    bottom: 48px;
+    ${from.mobileMedium} {
+        bottom: ${display === 'immersive' ? '3px' : '-3px'};
+        right: 5px;
+    }
+    ${from.mobileLandscape} {
+        right: 18px;
+    }
+    ${from.tablet} {
+        bottom: 3px;
+    }
+    ${from.desktop} {
+        display: none;
+    }
+`;
+
+const openExpandedMenu = (display: Display) => css`
```

Comment:
`openExpandedMenu` is an action, a doing thing, but this variable refers to some styles. Maybe `moreStyles`?